hookman/hookman_generator.py

```diff
@@ -127,13 +127,21 @@ def generate_plugin_template(self,
             dst_path: Path):
         """
         Generate a template with the necessary files and structure to create a plugin
+
+        A folder with the same name as the shared_lib_name argument will be created, with the following files:
             - config.yml
             - plugin.c
             - hook_specs.h
             - CMakeLists.txt
             - README.md
         """
-        plugin_folder = dst_path / plugin_name
+        if not shared_lib_name.isidentifier():
+            raise HookmanError("""
+            The shared libray name should be a valid identifier, with letters A through Z, underscore _
```

Comment:
"must be a valid Python identifier" is enough ("should" vs "must" is important). 😉

Reply:
✔️